packages/fxa-shared/metrics/glean/fxa-backend-metrics.yaml

```diff
@@ -1101,6 +1101,88 @@ inactive_account_deletion:
           The error code returned by the Cloud Tasks client.
         type: string
 
+  first_email_skipped:
```

Comment:
I was always under the impression that Glean metrics were related to user actions, these seem more like statsd/health.

Reply:
@vbudhram yeah I think that is still the right way to think about it.  This is not user action but we might need to know what happened in the process for each account.  Obviously I cannot do that with statsd.  The alternative is just plain logging, which is totally fine by me.  Since backend Glean write out to the log first, the main difference is the "free" Glean features we get after that.

Let me know if you feel strongly about this and I'll convert these events to plain logs.  Thanks.

Reply:
@chenba I think it is fine to keep as is, I don't have strong feelings either way.